.github/workflows/ci.yml

```diff
@@ -18,3 +18,25 @@ jobs:
         uses: docker://php:7.3-cli
         with:
           args: ./tools/php-cs-fixer fix --diff-format=udiff --dry-run --show-progress=dots --using-cache=no --verbose
+
+  type-checker:
+    name: Type Checker
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Update dependencies with composer
+        uses: docker://composer:1.9.0
```

Comment:
Composer docker image can have a PHP version different from the targeted one of the following step, so there can be a mismatch of the installed dependencies, right?

Reply:
Yes, that's definitely a problem we need to solve by providing dedicated Docker images!